stable/external-dns/templates/_helpers.tpl

```diff
@@ -0,0 +1,23 @@
+{{/* vim: set filetype=mustache: */}}
+{{/*
+Expand the name of the chart.
+*/}}
+{{- define "name" -}}
+{{- default .Chart.Name .Values.nameOverride | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+
+{{/*
+Create a default fully qualified app name.
+We truncate at 63 chars because some Kubernetes name fields are limited to this (by the DNS naming spec).
+*/}}
+{{- define "fullname" -}}
+{{- $name := default .Chart.Name .Values.nameOverride -}}
```

Comment:
I don't think there has been great consolidation on this, however when you have template helper methods and you include a chart as a dependency, these names actually get added to the same global space and collide. In the absence of a good "common chart" I would suggest prefixing these helper methods with the chart name or something else similarly unique so they don't have the potential to collide with one another. 

Ex. 
```
{{- define "external-dns.fullname" -}}
{{- define "external-dns.labels" -}}
```